### DISCUSSION TITLE
FF says is up to date, still get 404 error

### DISCUSSION THREAD

**Discussion by Susi-Susi** [FastForwardTeam/FastForward#662] [Q&A] [answered]
Hello, I'm aware you released a fix for this issue in #647 but I'm still getting the "you're not supposed to be here" 404 error. 

I've tried updating through extension manager in firefox and brave but both says definitions are up to date. 

Brave reports Version 0.2025 · Bypass definitions 4101474

and Firefox Version 0.2039 Bypass definitions 4101474

Are these the latest? Any ideas on how could I get it to work again?

Thanks.

**Comment by lostdusty** [comment, marked as answer]
This has been fixed already.
